addons/account_payment/tests/test_payment_refund_wizard.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_payment.tests.common import AccountPaymentCommon
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestPaymentRefundWizard(AccountPaymentCommon):
+
+    def setUp(self):
+        super().setUp()
+        self.admin_user = self.env.ref('base.user_admin')
+        self.demo_user = self.env['res.users'].search([('login', '=', 'demo')])
+
+    def test_amount_available_for_refund_consistency(self):
+        self.provider.support_refund = 'full_only'
+        tx = self._create_transaction('redirect', state='done')
+        tx._reconcile_after_done()
+        payment = tx.payment_id
+
+        wizard_admin = self.env['payment.refund.wizard'].with_user(self.admin_user).create({
+            'payment_id': payment.id,
+        })
```

Comment:
```suggestion
        wizard_admin = self.env['payment.refund.wizard'].with_user(self.admin_user).with_context(
            active_id=payment.id,
        ).new()
```

Difference between wizards and regular records is that wizards aren't stored in the database, so you shouldn't call `create` on them.

You need to look at how the fields are initialized to actually define them, for `payment_id`, it's set to the `active_id` context value: https://github.com/odoo/odoo/blob/fffc7c0f79a7e894c14d0fcdb27b507889887008/addons/account_payment/wizards/payment_refund_wizard.py#L15